CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
-# AWS IoT Over-the-air Update Library
+# Changelog for AWS IoT Over-the-air Update Library
+
+## v3.0.0 (February 2021)
+
+### Updates
+
+ - [#154](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/154) Updated `Ota_Shutdown` API. `Ota_Shutdown` now takes a parameter `unsubscribeFlag` which  indicates if unsubscribe operations should be performed from the job topics when shutdown is called.
+ - [#174](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/174), [#186](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/186) Adding support for different filetypes. `configOTA_FIRMWARE_UPDATE_FILE_TYPE_ID` config can be used to define the default firmware filetype id.
+### Other
+ - [#186](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/186) Adding OtaAppCallback for job and update completion.
+ - [#178](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/178), [#164](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/164), [#160](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/160) Updating custom job and active job logic flow.
+ - [#177](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/177) Adding example documentation for OTA APIs.
+ - [#155](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/155) Removing subscription to accepted topics i.e. "/jobs/$next/get/accepted"
```

Comment:
Can you add a period at the end of this for consistency?

Reply:
Added in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/188/commits/e8792c9f5707098718550e9c3b85b7e82696ef8e